src/__tests__/KeyPaths.types.test.ts

```diff
@@ -8,6 +9,6 @@ type NestedObject = {
   }
 }
 
-export function generatesKeyPathsFromObject(x: KeyPaths<NestedObject>): 'a' | 'b.b1' | 'b.b2' {
+export function generatesKeyPathsFromObject(x: Union.Diff<KeyPaths<NestedObject>, 'a' | 'b.b1' | 'b.b2'>): never {
```

Comment:
If I understand this correctly, we're asserting that the difference between the `KeyPaths<NestedObject>` and `'a' | 'b.b1' | 'b.b2'` is nothing. Seems reasonable to me 👍

Reply:
that's right! Before, I was only testing that `KeyPaths<NestedObject>` extends `'a' | 'b.b1' | 'b.b2'`. Without ts-toolbelt I could've added a second test function with swapped argument/return types.